src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -729,6 +729,12 @@ def compute_nhd_routing_v02(
                 qlat_sub = qlats.loc[param_df_sub.index]
                 q0_sub = q0.loc[param_df_sub.index]
 
+                param_df_sub = param_df_sub.reindex(
```

Comment:
Might open an issue to come back to this and clean up the indexing, but hacky enough for now...